tmt/utils.py

```diff
@@ -430,7 +430,7 @@ def write(self, path, data, level=2):
             return
         try:
             with open(path, 'w', encoding='utf-8', errors='replace') as target:
-                return target.write(data)
+                return target.write(repr(data))
```

Comment:
the reason is that data is tuple, when running the pytest. Not sure whats wrong yet. Applied this quick fix so far